shell/platform/embedder/embedder.h

```diff
@@ -141,6 +141,14 @@ typedef enum {
   kFlutterSemanticsFlagHasToggledState = 1 << 16,
   // If true, the semantics node is "on". If false, the semantics node is "off".
   kFlutterSemanticsFlagIsToggled = 1 << 17,
+  /// Whether the platform can scroll the semantics node when the user attempts
```

Comment:
Just two // here?

Reply:
And the Dart side for the summary :) Done.